docs/solr-cloud/solr-cloud-crd.md

```diff
@@ -543,3 +543,234 @@ spec:
+```
+      {
+        "name": "k8s-status",
+        "role": "k8s",
+        "collection": null,
+        "path": "/admin/collections"
+      },
+``` 
+In this case, the `"role":"k8s"` indicates the requesting user must be in the `k8s` role; notice that all default users have the `k8s` role.
+
+The Prometheus exporter sends GET requests to the `/admin/metrics` endpoint to collect metrics from each pod. 
+The exporter also hits the `/admin/ping` endpoint for every collection, which requires the following authorization rules:
+```
+      {
+        "name": "k8s-metrics",
+        "role": "k8s",
+        "collection": null,
+        "path": "/admin/metrics"
+      },
+      {
+        "name": "k8s-ping",
+        "role": "k8s",
+        "collection": "*",
+        "path": "/admin/ping"
+      },
+```
+The `"collection":"*"` setting indicates this path applies to all collections, which maps to endpoint `/collections/<COLL>/admin/ping` at runtime.
+
+### Option 2: User-provided Basic Auth Secret
+
+Alternatively, if users want full control over their cluster's security config, then they can provide a `kubernetes.io/basic-auth` secret containing the credentials for the user they want the operator to make API requests as:
+```
+spec:
+  ...
+  solrSecurity:
+    authenticationType: Basic
+    basicAuthSecret: user-provided-secret
+```
+The supplied secret must be of type [Basic Authentication Secret](https://kubernetes.io/docs/concepts/configuration/secret/#basic-authentication-secret) and define both a `username` and `password`.
+Here is an example of how to define a basic auth secret using YAML:
+```
+apiVersion: v1
+kind: Secret
+metadata:
+  name: my-basic-auth-secret
+type: kubernetes.io/basic-auth
+stringData:
+  username: k8s-oper
+  password: Test1234
+```
+With this config, the operator will make API requests to secured Solr pods as the `k8s-oper` user. 
+_Note: be sure to use a stronger password for real deployments_
+
+If users supply their own basic auth secret, then the operator *does not* bootstrap the `security.json`; 
+the reasoning is that if you're supplying your own basic auth credentials then you're also assuming the responsibility for configuring the desired access for this user.
+
+Users need to ensure their `security.json` contains the user supplied in the `basicAuthSecret` with read access to:
+```
+/admin/info/system
+/admin/collections
+/admin/metrics
+/admin/ping (for collection="*")
+```
+_Tip: see the authorization rules defined by the default `security.json` as a guide for configuring access for the operator user_
+
+### Prometheus Exporter with Basic Auth
+
+If you enable basic auth for your SolrCloud cluster, then you need to point the Prometheus exporter at the basic auth secret containing the credentials for making API requests to `/admin/metrics` and `/admin/ping` for all collections.
+
+```
+spec:
+  solrReference:
+    basicAuthSecret: user-provided-secret
+```
```

Comment:
I would re-iterate here the endpoints needed for the prometheus exporter auth user, when moving it to the prometheus exporter docs.